lib/render-code.js

```diff
@@ -0,0 +1,13 @@
+const prism = require('prismjs');
+require('prismjs/components/prism-json');
```

Comment:
I'd like this to have a comment, like:
```javascript
require('prismjs/components/prism-json'); // adds `json` to global `prism.languages` instance
```
because it is unusual that something is added globally. Normally you would do something like
```javascript
const prism = require('prismjs');
const prismJson = require('prismjs/components/prism-json');
prism.addLanguage(prismJson);
```